### PR TITLE
IDEM-370

### PR DESCRIPTION
**Pull request by JasminFarhi**
#370 IMG_7769

### CODE REVIEW

File: app/src/navigation/screens/ClaimScreen.tsx [hunk 1]
```diff
@@ -152,10 +151,20 @@ const ClaimScreen: React.FC = () => {
   };
   const openVerifyOtpScreen = async () => {
     setLoading(true);
-    const { countryCode, number } = formatMobileNumberState();
+
+    const newMobileState = formatMobileNumberState();
+    if (newMobileState.countryCode !== "+61") {
```
Comment (bitcoinbrisbane) [inline app/src/navigation/screens/ClaimScreen.tsx:156]: Sould we trim to be sure?

```suggestion
    if (newMobileState.countryCode.trim() !== "+61") {
```

Comment (JasminFarhi) [reply app/src/navigation/screens/ClaimScreen.tsx]: Added the trim in :)

**Review by bitcoinbrisbane — APPROVED** [review]

File: app/src/navigation/screens/ClaimScreen.tsx [hunk 2]
```diff
@@ -152,10 +151,20 @@ const ClaimScreen: React.FC = () => {
   };
   const openVerifyOtpScreen = async () => {
     setLoading(true);
-    const { countryCode, number } = formatMobileNumberState();
+
+    const newMobileState = formatMobileNumberState();
+    if (newMobileState.countryCode.trim() !== "+61") {
+      Alert.alert(
+        "Error",
+        "IDEM can only support Australian numbers for mobile verification"
```
Comment (Mitchell139) [inline app/src/navigation/screens/ClaimScreen.tsx:159]:
```suggestion
        "IDEM only supports Australian numbers for mobile claims/verification"
```

**Review by Mitchell139 — COMMENTED** [review]
I think the wording could be slightly improved

**Review by Mitchell139 — DISMISSED** [review]

File: app/src/navigation/screens/ClaimScreen.tsx [hunk 3]
```diff
@@ -152,10 +151,20 @@ const ClaimScreen: React.FC = () => {
...
+        "IDEM only supports Australian numbers for mobile claims/verification"
+      );
+      setLoading(false);
+      return;
+    }
+
     try {
       const otpResponse = await api.requestOtp({
-        mobileNumber: `${countryCode}${number}`
+        mobileNumber: `${newMobileState.countryCode}${newMobileState.number}`
```
Comment (pondpiu) [inline app/src/navigation/screens/ClaimScreen.tsx:167]: countryCode here still isn't getting trim

maybe we should trim in formatMobileNumberState function instead
so both line 156 and line 167 country code is trimmed

**Review by Mitchell139 — APPROVED** [review]

**Review by pondpiu — APPROVED** [review]

**Pull request merged**